packages/block-editor/src/components/block-popover/inbetween.js

```diff
@@ -28,6 +34,13 @@ function BlockPopoverInbetween( {
 	__unstableContentRef,
 	...props
 } ) {
+	// This is a temporary hack to get the inbetween inserter to recompute properly.
```

Comment:
Should we disable the respositioned sibling inserter and create a new one that renders visibly on every gap or to the side?